Physicist/Physicist/XML/Schemas/Types/ActorTypes.xsd

```diff
@@ -169,6 +169,11 @@
       <xs:element name="Actor" type="mstns:Actor"/>
     </xs:all>
     <xs:attribute name="class" type="xs:string" default="Player"/>
+    <xs:attribute name="rotateSound" type="xs:string" default="">
```

Comment:
This needs to be required else we need to provide a default with a name. Otherwise we have to pass null or empty strings as valid inside of the sound control flow.